swap_meet/clothing.py

```diff
@@ -1,2 +1,11 @@
-class Clothing:
-    pass
\ No newline at end of file
+from unicodedata import category
```

Comment:
This looks like an extraneous import. Possibly an auto-import? I recommend turning off that feature in VSCode